worldedit-core/src/main/java/com/sk89q/worldedit/session/PasteBuilder.java

```diff
@@ -46,6 +48,7 @@ public class PasteBuilder {
 
     private BlockVector3 to = BlockVector3.ZERO;
     private boolean ignoreAirBlocks;
+    private boolean ignoreStructureVoidBlocks = true; // default to true as it's their intended purpose
```

Comment:
I don't want to do this for API, at least not in 7.x. This changes existing behavior.